common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2972,9 +2973,9 @@ object CelebornConf extends Logging {
       .withAlternative("celeborn.slots.assign.extraSlots")
       .categories("master")
       .version("0.3.0")
-      .doc("Extra slots number when master assign slots.")
+      .doc("Extra slots number when master assign slots. Provided enough workers are available.")
       .intConf
-      .createWithDefault(100)
+      .createWithDefault(2)
```

Comment:
In my opinion, `celeborn.master.slot.assign.extraSlots` has the same meaning as `celeborn.master.slot.assign.minWorkers`; perhaps this configuration is sufficient.

Reply:
I think the main purpose is to revert the default value change of `celeborn.master.slot.assign.extraSlots`, which was from 2 to 100.